mk/board_qflight.mk

```diff
@@ -39,6 +39,9 @@ LIBOBJS += $(QFLIGHT_BUILD)/qflight_stub.o
 # Add missing parts from libc and libstdc++
 MISSING_TOOLCHAIN_FLAGS += -DHAVE_STD_NULLPTR_T=0 -DHAVE_STD_MOVE=0 -DHAVE_STD_REMOVE_REFERENCE=0 -DHAVE_TYPE_TRAITS_H=0 -DHAVE_BYTESWAP_H=0
 
+# Hardcoded libraries/AP_Common/missing/cmath defines in "make" to retain the current behavior
+EXTRAFLAGS += -DHAVE_CMATH_ISFINITE -DNEED_CMATH_ISFINITE_STD_NAMESPACE
```

Comment:
I'm not sure if this is true for qurt and we don't have easy access to the toolchain, AFAIR... :-/